pages/wellDone.js

```diff
@@ -0,0 +1,43 @@
+import Link from "next/link";
+import React from "react";
+import {
+  RoundButtonBack,
+  StyledContainer,
+  ImageContainer,
+} from "../components/StyledComponents";
+import {
+  BouncyImage1,
+  BouncyImage2,
+  BouncyImage3,
+} from "../components/StyledAnimation";
+
+export default function WellDonePage({ challenges }) {
+  // save the ghost attribut for all challenges with done=true
+
+  function showDoneGhosts() {
+    const doneGhosts = [];
+    challenges.map((challenge) => {
+      if (challenge.done === true) {
+        doneGhosts.push(challenge.ghosthappy);
+      }
+    });
+    return doneGhosts;
+  }
+
+  return (
+    <main>
+      <StyledContainer>
+        <h1>Well done!</h1>
+        <h2>Look at all the ghosts you helped to fight their fear!</h2>
+        <ImageContainer>
+          {showDoneGhosts().map((ghosthappy) => (
+            <BouncyImage1 key={ghosthappy} src={ghosthappy} alt="ghosthappy" />
```

Comment:
It's good to include the alt prop with a meaningful description for accessibility purposes. You could use something like "happy ghost" instead of "ghosthappy".